omero/developers/Web/Webclient.txt

```diff
@@ -0,0 +1,90 @@
+OMERO.webclient
```

Comment:
Could we have a more descriptive title? We already have a page titled 'Editing OMERO.web' for example